src/renderers/testing/__tests__/ReactTestRenderer-test.js

```diff
@@ -549,6 +553,124 @@ describe('ReactTestRenderer', () => {
     });
   });
 
+  it('toTree() handles simple components that return arrays', () => {
+    const Foo = ({children}) => children;
+
+    const renderer = ReactTestRenderer.create(
+      <Foo>
+        <div>One</div>
+        <div>Two</div>
+      </Foo>,
+    );
+
+    var tree = renderer.toTree();
+
+    cleanNodeOrArray(tree);
+
+    expect(prettyFormat(tree)).toEqual(
```

Comment:
Could you explain why `prettyFormat` is necessary here?

Reply:
`prettyFormat` isn't necessary. I was simply following the convention with the other `toTree` tests that use `prettyFormat`.

Reply:
I ran the tests again, removing all usages of `prettyFormat` and everything is 👌

Reply:
when I originally wrote these tests, `prettyFormat` was strangely required. I forget why, but it seems like there was some strange bug in the deep equality algorithm which was causing things to fail (i don't remember at this time what they were). If we can remove it now and things work, that sounds good to me!